line 59 edit from provisioner to builder

DevOpsBoondoggles opened the pull request:
links to builder but says provisioner.  I can't see a provisioner for Azure so assume this is just Typo

azurecla commented:
Hi __@kevinlam1__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (klam). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

cjgronlund commented:
@kevinlam1 
Good catch! Thank you for your contribution.
--Carolyn

cjgronlund commented:
@ggailey777 
It was an obvious typo, so I accepted.

DevOpsBoondoggles committed:
line 59 edit from provisioner to builder

links to builder but says provisioner.  I can't see a provisioner for Azure so assume this is just Typo
